ups/pipe_supertask.table

```diff
@@ -1,4 +1,5 @@
 setupRequired(pipe_base)
+setupRequired(pipe_tasks)
```

Comment:
I intentionally left pipe_tasks out since there was this plan to have a registry with all the tasks, but I guess for now its ok, `pipe_tasks` should be enough though